app.js

```diff
@@ -141,6 +147,32 @@ export default class App extends Component {
     return viewRender
   }
 
+  previousImage() {
```

Comment:
Move the responsibility of managing the practice session into the `CardCarousel` component. It can have its own local state.